CONTRIBUTING.rst

```diff
@@ -18,12 +18,38 @@ first in an appropriate venue, drafting a PEP and gathering feedback, and
 developing at least a prototype reference implementation of your idea.
 
 
+Contributing changes to existing PEPs
+-------------------------------------
+
+In general, most non-Draft/Active PEPs are considered to be historical
+documents rather than living specifications or documentation. Major changes to
+their core content usually require a new PEP, while smaller modifications may
+or may not be appropriate, depending on the PEP's status. See `PEP 1
+<https://www.python.org/dev/peps/pep-0001/#pep-maintenance>`__ for more.
+
+Copyediting and proofreading Draft and Active PEPs (including fixing spelling
+and formatting issues) is welcomed (subject to review by the PEP authors),
+and can be done via pull request to this repo, though we generally advise
+against PRs that simply mass-correct minor typos on older PEPs which don't
+significantly impair meaning and understanding.
+Substantive content changes should first be proposed on PEP discussion threads.
+
+If you're still unsure, we encourage you to reach out first before opening a
+PR here. For example, you could contact the PEP author(s), propose your idea in
+a discussion venue appropriate to the PEP (such as `Typing-SIG
+<https://mail.python.org/archives/list/typing-sig@python.org/>`__ for static
+typing, or `Packaging Discourse <https://discuss.python.org/c/packaging/>`__
+for packaging), or `open an issue <https://github.com/python/peps/issues>`__.
+
+
 Commit messages and PR titles
 -----------------------------
 
-When adding or modifying a PEP, please always include the PEP number in the
-commit summary and pull request title.
-For example, ``PEP NNN: <summary of changes>``.
+When adding or modifying a PEP, please include the PEP number in the commit
+summary and pull request title. For example, ``PEP NNN: <summary of changes>``.
+Likewise, prefix rendering infrastructure changes with ``Infra:``, linting
+alterations with ``Lint:`` and other non-PEP meta changes, such as updates to
+the Readme/Contributing Guide, issue/PR template, etc., with ``Meta:``.
```

Comment:
That was the convention I was following before, and I'm okay with that if others agree (provided we _do_ ensure that changes to specific PEPs are prefixed properly).

I don't think its necessary to be too prescriptive on this either (we have GitHub tags now anyway); this mostly was to avoid individual PEP editors coming up with their own cryptic, bespoke initialisms and confusing multiple other PEP editors, but that seems to be a pretty limited and specific problem that is better solved in other more appropriate ways.

```suggestion
```